### ISSUE TITLE
Adding Aqara Zigbee devices

### ISSUE THREAD

**Issue by coradia** [home-assistant/home-assistant.github.io#35100]
### Feedback

Can you please also include information about the Zigbee option to connect devices? Appreciate this uses the normal Zigbee integration approach but would expect to see on this page at least a reference to that but more so what products are supported.

### URL

https://www.home-assistant.io/integrations/aqara/

### Version

2024.10.1

**Comment by Hedda** [comment]
Yes further clarification and a device list with compatibility would be nice since Aqara sells both Zigbee devices and Matter devices. 

Their Zigbee devices can work Home Assistant's built-in native ZHA integration -> https://www.home-assistant.io/integrations/zha/

Their Matter devices can work with Home Assistant's built-in native Matter integrations -> https://www.home-assistant.io/integrations/matter/

Further note Aqara also sells IP cameras that might be compatible with the Camera integration -> https://www.home-assistant.io/integrations/camera/

All or most of their devices are probably also be compatible with Aqara and/or Xiaomi branded gateways (propriatory hubs) -> https://www.home-assistant.io/integrations/xiaomi_aqara/

**Comment by coradia** [comment]
Interesting that all references to Zigbee have been stripped from the page.

**Comment by Hedda** [comment]
> Interesting that all references to Zigbee have been stripped from the page.

Maybe related to the fact that Aqara has now joined the Works with Home Assistant partnership program for Matter specifically?

- https://www.home-assistant.io/blog/2024/09/03/aqara-joins-works-with-home-assistant/

- https://partner.home-assistant.io/
